src/coreclr/jit/lsra.cpp

```diff
@@ -836,6 +837,28 @@ LinearScan::LinearScan(Compiler* theCompiler)
     rbmIntCalleeTrash = compiler->rbmIntCalleeTrash;
     regIntLast        = compiler->regIntLast;
     isApxSupported    = compiler->canUseApxEncoding();
+    if (isApxSupported)
+    {
+        regIndices = new regNumber[ACTUAL_REG_COUNT];
+        for (int i = 0; i < (int)ACTUAL_REG_COUNT; i++)
+        {
+            regIndices[i] = (regNumber)i;
+        }
+    }
+    else
+    {
+        regIndices =
+            new regNumber[]{(regNumber)0,  (regNumber)1,  (regNumber)2,  (regNumber)3,  (regNumber)4,  (regNumber)5,
```

Comment:
i think you should use the allocator to allocate this using `new (theCompiler->getAllocator(CMK_LSRA_Interval))` or equivalent